Source/Diagnostics/ReducedDiags/ParticleHistogram2D.cpp

```diff
@@ -298,7 +298,7 @@ void ParticleHistogram2D::WriteToFile (int step) const
     data.setPosition<amrex::Real>({0.5, 0.5});
 
     auto dataset = io::Dataset(
-            io::determineDatatype<double>(),
+            io::determineDatatype<amrex::Real>(),
```

Comment:
Wouldn't the properties be in `ParticleReal`?
```suggestion
            io::determineDatatype<amrex::ParticleReal>(),
```

Reply:
Yes, you are right!